worker_tasks/feed_scraper.py

```diff
@@ -17,12 +19,16 @@
 celery = Celery(app.name, broker=app.config['CELERY_BROKER_URL'])
 celery.conf.update(app.config)
 
+
+def get_full_text(url):
+    html = requests.get(url).content
+    return Document(html).summary()
```

Comment:
You can look at the source code for summary here: https://github.com/buriy/python-readability/blob/0c2f29ed0df927e4305331f7411a5a1efe2c9cbc/readability/readability.py

It's what python-readability calls it's main function which gets the best candidate for the article and parses it down to the relevant tags.